gooddata-sdk/gooddata_sdk/catalog_model/data_source_content_objects/__init__.py

```diff
@@ -0,0 +1,3 @@
+# (C) 2022 GoodData Corporation
+
+from gooddata_sdk.catalog_model.data_source_content_objects.table import CatalogDataSourceTable
```

Comment:
Any benefit of having this import?

Reply:
completely refactored, inner imports removed